### DISCUSSION TITLE
Question about specific clearance calculation

### DISCUSSION THREAD

**Discussion by Sawy25** [Open-Systems-Pharmacology/Forum#1442] [Questions & Problems] [answered]
Hi all,

When we define renal plasma clearance, we define it as per body weight. I wonder if we need to input the body weight as well for the simulated subject because by default it's 73 kg. But if this is the case we need to update also the other parameters (e.g. kidney volume, blood flow...) for this specific individual because all of them contribute to specific clearance calculations? Also, it means that for different simulated individuals, we need different profiles for the same compound each adjusted for patient biometric data!

![image](https://user-images.githubusercontent.com/115218843/213429161-0728209c-9ab1-4b04-83f5-e49a78193d69.png)

Thanks

**Comment by Sawy25** [comment]
Just showing this example, if someone can help me with this.

I am simulating a drug with a renal clearance of 7.15 ml/min/kg in a population with a mean weight of 57. The specific clearance would be different based on the weight used for its calculation and accordingly getting different simulated profiled. 

![image](https://user-images.githubusercontent.com/115218843/214546181-225a97a1-b15e-4ac6-8650-19f4d241838c.png)

![image](https://user-images.githubusercontent.com/115218843/214545989-9f4ec918-0466-4a77-b6a2-72ae22eb9e53.png)

![image](https://user-images.githubusercontent.com/115218843/214546258-be231743-0e6b-4344-bbdc-337c7e0c6685.png)

![image](https://user-images.githubusercontent.com/115218843/214546322-1e1977ae-3f56-4e3d-87a4-081408209b5f.png)


So the question is what is the right way to do this? do I need to add different clearance profiles for the same drug based on the simulated population's body weight? 

The specific clearance unit is 1/time, so is it supposed to be calculated as a standard measurement for drug clearance for a patient weight 73 kg, age 30 and not to be played with?

Many thanks
Salma

**Comment by tobiasK2001** [comment]
Dear Salma,

sorry for the slow reply and I am not sure if my answer is still relevant, but the specific clearance is calculated from your input as plasma clearance according to this formula: 
![image](https://user-images.githubusercontent.com/25585248/221907064-5f7a561a-2dd4-4fc3-9bcd-29c558f965fa.png)
So basically the plasma clearance is back-calculated to an intrinsic kidney CL and normalized to the kidney volume.

PK-Sim uses the specific clearance for calculation in populations. So If you would exchange your Individual with a virtual population PK-sim will use e.g. 2.22 1/min and multiply it with the kidney volume in each virtual individual. That way from the same specific CL each Individual will get a different plasma CL according to the individual body weight and Kidney volume.

Based on that, if you think your plasma CL is based totally on Kidney clearance, you might think of refining your specific kidney CL to your measured data and use the refined value for extrapolation to populations.

Hope that might help you.

Best, Tobias

**Comment by Sawy25** [comment, marked as answer]
Thanks a lot, Tobias for the clarification. It makes sense now!

Kind regards
Salma

**Comment by songsong02** [comment]
I'm very sorry, but I may not have fully understood your response. For the specific clearance used in the simulation in the screenshot, was it calculated based on the 73.00kg input caculation parameters Body weights, or based on the 57kg in the individuals module?
<img width="927" height="585" alt="image" src="https://github.com/user-attachments/assets/041b5195-1349-4430-9dd5-d8c132c70eb7" />
Looking forward to your reply!
